pages/admin/organisations/index.tsx

```diff
@@ -0,0 +1,32 @@
+import Organisations from "../../../components/containers/Admin/Organisations/Organisations";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { requireAuthentication } from "../../../lib/auth";
+import { crudOrganisations } from "@lib/dataLayer";
+
+export const getServerSideProps = requireAuthentication(async (context) => {
+  {
+    // getStaticProps is serverside, and therefore instead of doing a request,
+    // we import the invoke Lambda function directly
+    const lambdaResult = await crudOrganisations({ method: "GET" });
+
+    const organisations =
```

Comment:
There's a nifty new ES feature that can reduce the complexity of this type of call.
It's also a lot less of a pain to write out...

`const organizations = lambdaResult?.data?.records? ?? []`